### PR TITLE
plugins/ocp: Add OCP Telemetry String log page, Telemetry log page

### PR DESCRIPTION
**Pull request by Arunpandian15**
Added OCP 2.5 Telemetry string log page and Telemetry log page support.

### CODE REVIEW

**Comment by igaw** [comment]
@arthurshau

File: plugins/ocp/ocp-nvme.c [hunk 1]
```diff
@@ -2100,6 +2304,650 @@ static int get_plp_health_check_interval(int argc, char **argv, struct command *
...
+{
+    return d_raw((unsigned char *)log_data_buf, total_log_page_size);
+}
+
+static int get_c9_log_page(struct nvme_dev *dev, char *format)
+{
+    int ret = 0;
+    int fmt = -1;
+    __u8 *header_data;
+    __u8 *full_log_buf_data;
```
Comment (arthurshau) [inline plugins/ocp/ocp-nvme.c:2839]: nit: `__u8 *full_log_buf_data = NULL;` to avoid -Wsometimes-uninitialized

**Comment by arthurshau** [comment]
One minor nit, other than that LGTM.

File: plugins/ocp/ocp-nvme.c (hunk 1 above)
Comment (Arunpandian15) [reply plugins/ocp/ocp-nvme.c]: Resolved. Thanks @arthurshau

**Comment by Arunpandian15** [comment]
@igaw, @arthurshau Addressed the review comments. Please check

**Comment by Arunpandian15** [comment]
> gcc is unhappy:
> 
> ```shell
> ../plugins/ocp/ocp-nvme.c: In function ‘ocp_print_C9_log_normal’:
> ../plugins/ocp/ocp-nvme.c:2468:41: error: iteration 15 invokes undefined behavior [-Werror=aggressive-loop-optimizations]
>  2468 |         printf("%d", log_data->reserved1[j]);
>       |                      ~~~~~~~~~~~~~~~~~~~^~~
> ../plugins/ocp/ocp-nvme.c:2467:19: note: within this loop
>  2467 |     for (j = 0; j < 16; j++)
>       |                 ~~^~~~../plugins/ocp/ocp-nvme.c: In function ‘ocp_print_C9_log_normal’:
> ../plugins/ocp/ocp-nvme.c:2468:41: error: iteration 15 invokes undefined behavior [-Werror=aggressive-loop-optimizations]
>  2468 |         printf("%d", log_data->reserved1[j]);
>       |                      ~~~~~~~~~~~~~~~~~~~^~~
> ../plugins/ocp/ocp-nvme.c:2467:19: note: within this loop
>  2467 |     for (j = 0; j < 16; j++)
>       |                 ~~^~~~
> ```

@igaw Resolved. Please check

**Comment by igaw** [comment]
Thanks!

**Pull request merged**